queue_services/entity-emailer/src/entity_emailer/email_processors/dissolution_notification.py

```diff
@@ -0,0 +1,245 @@
+
+
+def _get_pdfs(
+        status: str,
+        token: str,
+        business: dict,
+        filing: Filing,
+        filing_date_time: str,
+        effective_date: str) -> list:
+    # pylint: disable=too-many-locals, too-many-branches, too-many-statements, too-many-arguments
+    """Get the pdfs for the incorporation output."""
+    pdfs = []
+    attach_order = 1
+    headers = {
+        'Accept': 'application/pdf',
+        'Authorization': f'Bearer {token}'
+    }
+    legal_type = business.get('legalType', None)
+
+    if status == Filing.Status.PAID.value:
+        # add filing pdf
+        filing_pdf = requests.get(
+            f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}',
+            headers=headers
+        )
+        if filing_pdf.status_code != HTTPStatus.OK:
+            logger.error('Failed to get pdf for filing: %s', filing.id)
+            capture_message(f'Email Queue: filing id={filing.id}, error=pdf generation', level='error')
+        else:
+            filing_pdf_encoded = base64.b64encode(filing_pdf.content)
+            pdfs.append(
+                {
+                    'fileName': 'Voluntary Dissolution Application.pdf',
+                    'fileBytes': filing_pdf_encoded.decode('utf-8'),
+                    'fileUrl': '',
+                    'attachOrder': attach_order
+                }
+            )
+            attach_order += 1
+
+        corp_name = business.get('legalName')
+        business_data = Business.find_by_internal_id(filing.business_id)
+        receipt = requests.post(
+            f'{current_app.config.get("PAY_API_URL")}/{filing.payment_token}/receipts',
+            json={
+                'corpName': corp_name,
+                'filingDateTime': filing_date_time,
+                'effectiveDateTime': effective_date if effective_date != filing_date_time else '',
+                'filingIdentifier': str(filing.id),
+                'businessNumber': business_data.tax_id if business_data and business_data.tax_id else ''
+            },
+            headers=headers
+        )
+        if receipt.status_code != HTTPStatus.CREATED:
+            logger.error('Failed to get receipt pdf for filing: %s', filing.id)
+            capture_message(f'Email Queue: filing id={filing.id}, error=receipt generation', level='error')
+        else:
+            receipt_encoded = base64.b64encode(receipt.content)
+            pdfs.append(
+                {
+                    'fileName': 'Receipt.pdf',
+                    'fileBytes': receipt_encoded.decode('utf-8'),
+                    'fileUrl': '',
+                    'attachOrder': attach_order
+                }
+            )
+            attach_order += 1
+    elif status == Filing.Status.COMPLETED.value:
+        # add certificateOfDissolution
+        certificate = requests.get(
+            f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}'
```

Comment:
I think all docs should point to the v2 version URL now that filings UI document section is pointing to v2 service.